lib/ldp.js

```diff
@@ -53,8 +53,8 @@ function LDP(argv) {
   }
 
   ldp.pathFilter = regexp().start(ldp.mount).toRegExp();
-  ldp.xssProxy = argv.xssProxy;
-  ldp.proxyFilter = regexp().start(ldp.xssProxy).toRegExp();
+  ldp.proxy = argv.proxy;
+  ldp.proxyFilter = regexp().start(ldp.proxy).toRegExp();
```

Comment:
Quick question: what does proxyFilter do, and why do we need a filter for proxy?